### PR TITLE
CircleCI checker

### PR DESCRIPTION
**Pull request by sr229**
You need to list your project in [CircleCI](https://circleci.com) for this to work

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: node:7.4.0
```
Comment (sam6321) [inline circle.yml]: Need node 8.1.x. unless this is an eslint specific requirement for 7.4.0

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      - run:
+          name: Install dependencies
+          command: npm i -g eslint && npm i --save
+      - run:
+          name: ESLint Check
+          command: npm run lint
+notify:
+  webhooks:
+  # you need a custom webhook for this. This one posts to Clarity's Discord
+  #  - url: https://claraio-circledisc.herokuapp.com/hooks/circleci
```
Comment (sam6321) [inline circle.yml]: I'll set this up.

File: circle.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      - run:
+          name: Install dependencies
+          command: npm i -g eslint && npm i --save
+      - run:
+          name: ESLint Check
+          command: npm run lint
+# notify:
+#  webhooks:
+  # you need a custom webhook for this. This one posts to Clarity's Discord
+  #  - url: https://claraio-circledisc.herokuapp.com/hooks/circleci
```
Comment (sam6321) [inline circle.yml:17]: I'll set this up.

File: circle.yml (hunk 1 above)
Comment (sr229) [reply circle.yml]: @sam6321 Because of the issues present in the newer NPM, I do not recommend using the newer Node.

**Comment by sr229** [comment]
Superseded by #6

**Pull request closed without merging**